web/client/src/components/form/TokenCredentials.jsx

```diff
@@ -7,35 +7,26 @@ import {
   Radio,
   FormControlLabel,
   Button,
-  Switch,
-  TextField as TextFieldMUI,
 } from "@material-ui/core";
 import { Alert } from "@material-ui/lab";
 import { TextField } from "formik-material-ui";
 import PropTypes from "prop-types";
 import "../../styles/form.css";
 
 /**
- * @param {Object} props contains setFieldValue function for updating credentials field manually
+ * @param {Object} props contains setFieldValue function for updating credential field manually
  * @return {Box} containing form fields for adding scopes
  */
 export default function TokenCredentials(props) {
   const [credFormat, setCredFormat] = useState("file");
   const [error, setError] = useState(false);
   const [errorMsg, setErrorMsg] = useState("");
   const [success, setSuccess] = useState(false);
-  const [saveCredential, setSaveCredential] = useState(false);
 
-  const { setFieldValue, tokenAvailable, parsedCredential } = props;
+  const { setFieldValue } = props;
```

Comment:
This change seems like a plausible alternative, however , it would essentially delete the caching ability of the web app, right?  I'm not sure if that would be good.